src/cityreader/cityreader.py

```diff
@@ -1,30 +1,58 @@
-# Create a class to hold a city location. Call the class "City". It should have
-# fields for name, lat and lon (representing latitude and longitude).
+import csv
 
+#tried to do pandas >_>
+# import pandas
+# df = pandas.read_csv('cities.csv')
+# print(df)
 
+# Create a class to hold a city location. Call the class "City". It should have
+# fields for name, lat and lon (representing latitude and longitude).
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+    def __str__(self):    
+        return f'{self.name}, {self.lat}, {self.lon}'    
 # We have a collection of US cities with population over 750,000 stored in the
 # file "cities.csv". (CSV stands for "comma-separated values".)
-#
 # In the body of the `cityreader` function, use Python's built-in "csv" module 
 # to read this file so that each record is imported into a City instance. Then
 # return the list with all the City instances from the function.
 # Google "python 3 csv" for references and use your Google-fu for other examples.
-#
 # Store the instances in the "cities" list, below.
-#
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
+  # TODO 
+  # Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+
+    #working solution!----------------------------------------------------
+    with open("cities.csv") as csvfile:
+        for city in list(csv.reader(csvfile))[1:]:
+            #when append, the float selection is location of object in list and because they
+            #are integers 
+            cities.append(City(city[0], float(city[3]), float(city[4])))
     return cities
+    #---------------------------------------------------------------------
 
-cityreader(cities)
+    #failed solution---------------
```

Comment:
Hhahahah, at least you tried. Proud of you. Check out my repo a little later on if you want a solution; I think some of the others in the small group also succeeded at this. I might ask them to give a small demo of it!